pyproject.toml

```diff
@@ -14,9 +14,10 @@ include = [ { path = "tests", format = "sdist" } ]
 [tool.poetry.dependencies]
 python = ">=3.8"
 RapidSMS = ">=2.0"
```

Comment:
`RapidSMS` might need to be upgraded, it seems to be crashing this build: https://github.com/caktus/rapidsms-smpp-gateway/actions/runs/5581453954/jobs/10199938046?pr=16#step:6:67